tests/plugins/dialog/plugin.js

```diff
@@ -81,6 +81,16 @@
 	bender.editor = {};
 
 	bender.test( {
+		tearDown: function() {
+			var dialog = CKEDITOR.dialog.getCurrent();
```

Comment:
Since this tearDown looks the same as the one used in another test (`positioning`) and we already have `_helpers` for dialog tests, maybe we could move this code to a separate function with proper naming, like `hideAllDialogs`?